polygon-digital-carbon/src/utils/MethodologyCategories.ts

```diff
@@ -0,0 +1,217 @@
+    map.set('AM0044', 'Energy Efficiency')
+    map.set('AM0046', 'Energy Efficiency')
+    map.set('AM0055', 'Energy Efficiency')
+    map.set('AM0055', 'Energy Efficiency')
+    map.set('AM0056', 'Energy Efficiency')
+    map.set('AM0061', 'Energy Efficiency')
+    map.set('AM0062', 'Energy Efficiency')
+    map.set('AM0067', 'Energy Efficiency')
+    map.set('AM0091', 'Energy Efficiency')
+    map.set('AM0105', 'Energy Efficiency')
+    map.set('AM0106', 'Energy Efficiency')
+    map.set('AM0113', 'Energy Efficiency')
+    map.set('AM0114', 'Energy Efficiency')
+    map.set('AM0115', 'Energy Efficiency')
+    map.set('AM0116', 'Energy Efficiency')
+    map.set('AM0117', 'Energy Efficiency')
+    map.set('AM0118', 'Energy Efficiency')
+    map.set('AM0119', 'Energy Efficiency')
+    map.set('AM0120', 'Energy Efficiency')
+    map.set('AM0121', 'Energy Efficiency')
+    map.set('VM0008', 'Energy Efficiency')
+    map.set('VM0018', 'Energy Efficiency')
+    map.set('VM0019', 'Energy Efficiency')
+    map.set('VM0020', 'Energy Efficiency')
+    map.set('VM0025', 'Energy Efficiency')
+    map.set('VMR0005', 'Energy Efficiency')
+    map.set('VMR0006', 'Energy Efficiency')
+    map.set('VM0001', 'Energy Efficiency')
+    map.set('VM0002', 'Energy Efficiency')
+    map.set('VM0008', 'Energy Efficiency')
+    map.set('VM0019', 'Energy Efficiency')
+    map.set('VM0020', 'Energy Efficiency')
+    map.set('VM0018', 'Energy Efficiency')
+    map.set('ACM0012', 'Energy Efficiency')
+    map.set('AMS-III.Z', 'Energy Efficiency')
+    map.set('GS TPDDTEC v 1.', 'Energy Efficiency')
+    map.set('AMS-II.G.', 'Energy Efficiency')
+
+    //Agriculture
+    map.set('VM0041', 'Agriculture')
+    map.set('VM0042', 'Agriculture')
+    map.set('VM0017', 'Agriculture')
+    map.set('VMR0003', 'Agriculture')
+    map.set('AMS-III.AU.', 'Agriculture')
+    map.set('AMS-III.BE', 'Agriculture')
+    map.set('AMS-III.BF', 'Agriculture')
+    map.set('VM0017', 'Agriculture')
+    map.set('AMS-III.D.', 'Agriculture')
+
+    //Forestry
+    map.set('VM0004', 'Forestry')
+    map.set('VM0007', 'Forestry')
+    map.set('VM0024', 'Forestry')
+    map.set('VM0027', 'Forestry')
+    map.set('VM0033', 'Forestry')
+    map.set('VM0036', 'Forestry')
+    map.set('VM0003', 'Forestry')
+    map.set('VM0004', 'Forestry')
+    map.set('VM0005', 'Forestry')
+    map.set('VM0006', 'Forestry')
+    map.set('VM0007', 'Forestry')
+    map.set('VM0009', 'Forestry')
+    map.set('VM0010', 'Forestry')
+    map.set('VM0011', 'Forestry')
+    map.set('VM0012', 'Forestry')
+    map.set('VM0015', 'Forestry')
+    map.set('VM0029', 'Forestry')
+    map.set('VM0034', 'Forestry')
+    map.set('VM0035', 'Forestry')
+    map.set('VM0037', 'Forestry')
+    map.set('AR-ACM0003', 'Forestry')
+    map.set('AR-AM0014', 'Forestry')
+    map.set('AR-AMS0003', 'Forestry')
+    map.set('AR-AMS0007', 'Forestry')
+    map.set('AR-ACM0001', 'Forestry')
+    map.set('AM0014', 'Forestry')
```

Comment:
This methodology is improperly labeled `Forestry` - it got confused because of the similarly named "AR-AM0014"

The correct category for AM0014 is "Energy Efficiency"